README.md

```diff
@@ -50,18 +50,34 @@ The back end is a node.js server built with [express](https://expressjs.com/), [
   - ```bash
     docker-compose run --rm --entrypoint='yarn test' ssmp-server
     ```
-    
-### Connecting to Phenotips 
+
+### Connecting to Phenotips
 
 The Phenotips variant matching endpoint specified [here](https://github.com/ccmbioinfo/report-scripts/blob/master/phenotips-api.md#matching-endpoint) can only be reached directly from our staging virtual machine ubuntu@ssmp-dev in the [CHEO-RI tenancy](https://github.com/ccmbioinfo/cheo-ri-infrastructure).
 
-In development, for the Express server to successfully make a call to the endpoint, we need to use SSH's local forwarding protocol to forward a port from your local machine to the staging server machine. The SSH client listens to any requests or connections on a configured port, and when it receives a connection, it tunnels the connection to the SSH server. 
+In development, for the Express server to successfully make a call to the endpoint, we need to use SSH's local forwarding protocol to forward a port from your local machine to the staging server machine. The SSH client listens to any requests or connections on a configured port, and when it receives a connection, it tunnels the connection to the SSH server.
+
+If you are using CCM's VMs, you can set up local forwarding as follows:
+
+- To login into your VM: `eval $(ssh-agent -s) && ssh-add` and `ssh -A <username>@dev-<username>.ccm.sickkids.ca`. Make sure that you already have a VM allocated to you.
+- To forward your local port to ubuntu@ssmp-dev: `ssh -ANL 0.0.0.0:8443:dev.phenotips.genomics4rd.ca:443 ubuntu@ssmp.ccmdev.ca`. For now, this command would need to be run alongside `docker-compose up` when you want to bring up the app.
+- Set `G4RD_URL` in your local `.env` to `https://dev-<username>.ccm.sickkids.ca:8443`.
+  Now, any request sent to your `G4RD_URL` would be tunneled to `dev.phenotips.genomics4rd.ca:443` on the staging VM.
+
+### Building the remote test server
 
-If you are using CCM's VMs, you can set up local forwarding as follows: 
-  - To login into your VM: ```eval $(ssh-agent -s) && ssh-add``` and ```ssh -A <username>@dev-<username>.ccm.sickkids.ca```. Make sure that you already have a VM allocated to you. 
-  - To forward your local port to ubuntu@ssmp-dev: ```ssh -ANL 0.0.0.0:8443:dev.phenotips.genomics4rd.ca:443 ubuntu@ssmp.ccmdev.ca```. For now, this command would need to be run alongside `docker-compose up` when you want to bring up the app.
-  - Set ```G4RD_URL``` in your local ```.env``` to ```https://dev-<username>.ccm.sickkids.ca:8443```.
-Now, any request sent to your ```G4RD_URL``` would be tunneled to ```dev.phenotips.genomics4rd.ca:443``` on the staging VM. 
+Apart from Phenotips, another data source for the SSMP development instance is a Node/Express server that queries a MySQL database that has been populated wtih variants from the STAGER application database.
```

Comment:
@jennyziyi-xu Can you upload the dummy Stager data to our Microsoft Teams folder and link it here? 

Also, it might be helpful to document the `docker exec` command you ran to load the SQL script into MySQL.

Reply:
@hannahle I think you have previously uploaded the dummy Stager data: [link ](https://sickkidsca.sharepoint.com/:u:/r/sites/thecenterforcomputationalmedicineworkspace/Shared%20Documents/SSMP/data/stager-local-20210716.sql?csf=1&web=1&e=dJy8Ao). I guess you mean to move it to another folder?

Reply:
Ah, I forgot I did that. You can use the same link.

Reply:
Got it. I've added the `docker exec` command as an alternative to using MySQLWorkbench.